Kickstarter-iOS/Views/Controllers/CreatePasswordViewController.swift

```diff
@@ -64,26 +152,36 @@ final class CreatePasswordViewController: UITableViewController {
   }
 
   override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-    let row = CreatePasswordRow.allCases[indexPath.row]
     let cell = tableView.dequeueReusableCell(withClass: SettingsTextInputCell.self, for: indexPath)
 
     guard let textInputCell = cell as? SettingsTextInputCell else { return cell }
 
-    textInputCell.configure(with: row.title, placeholder: row.placeholder)
+    let row = CreatePasswordRow.allCases[indexPath.row]
+
+    textInputCell.configure(with: row.placeholder, returnKeyType: row.returnKeyType, title: row.title)
+    textInputCell.configure(with: textFieldTargetActions(for: self, row: row))
+
     return textInputCell
   }
 
   // MARK: - UITableViewDelegate
 
+  override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+    guard let textInputCell = tableView.cellForRow(at: indexPath) as? SettingsTextInputCell else { return }
+
+    self.viewModel.inputs.textFieldShouldBecomeFirstResponder(textInputCell.textField)
```

Comment:
I think we generally try to keep view models free of any knowledge of `UIKit` things. Could we pass the index path's row here instead to determine which text field needs to become the first responder?